pom.xml

```diff
@@ -34,26 +34,26 @@
   </licenses>
 
   <properties>
-    <rmb.version>34.0.0</rmb.version>
-    <folio-di-support.version>1.6.0</folio-di-support.version>
-    <folio-service-tools.version>1.9.0</folio-service-tools.version>
-    <folio-holdingsiq-client.version>2.1.0</folio-holdingsiq-client.version>
-    <folio-liquibase-util.version>1.5.0</folio-liquibase-util.version>
-    <mod-configuration-client.version>5.7.8</mod-configuration-client.version>
-
-    <vertx.version>4.3.1</vertx.version>
+    <rmb.version>35.0.0</rmb.version>
+    <folio-di-support.version>1.7.0-SNAPSHOT</folio-di-support.version>
+    <folio-service-tools.version>1.10.0-SNAPSHOT</folio-service-tools.version>
+    <folio-holdingsiq-client.version>2.2.0-SNAPSHOT</folio-holdingsiq-client.version>
+    <folio-liquibase-util.version>1.6.0-SNAPSHOT</folio-liquibase-util.version>
+    <mod-configuration-client.version>5.9.0-SNAPSHOT</mod-configuration-client.version>
+
+    <vertx.version>4.3.3</vertx.version>
     <aspectj.version>1.9.9.1</aspectj.version>
-    <spring.version>5.3.21</spring.version>
-    <jackson.version>2.13.3</jackson.version>
-    <postgresql.version>42.4.1</postgresql.version>
+    <spring.version>5.3.23</spring.version>
+    <jackson.version>2.14.0-rc1</jackson.version>
```

Comment:
Release candidates should not be part of dependecies

Reply:
Got it. took version 2.13.4